src/app/routes/liveRadio/getInitialData/index.js

```diff
@@ -69,7 +40,7 @@ export default async ({ path: pathname, pageType, service }) => {
           pageDataPromise,
           service,
           path: pathname,
-          radioService: getRadioService({ service, pathname }),
+          radioService: getRadioService(service),
```

Comment:
I _think_ we still need the service for `persian` to be `persian` when the path is `bbc_persian_radio`, as in [this line](https://github.com/bbc/simorgh/pull/7427/files#diff-fb5c8fb7e089a8f07c978ffdea6dd565L51). The [new getRadioService function](https://github.com/bbc/simorgh/pull/7427/files#diff-be61ccec88e424608cbb0856f441287fR1-R6) would always return `dari`.

Reply:
Correct